scrambler-interface/WebContent/scrambler-interface/js/ui.js

```diff
@@ -542,7 +543,8 @@ var mark2 = {};
                     round: round,
                     groupCount: numGroups,
                     scrambleCount: numSolves,
-                    extraScrambleCount: numExtraSolves
+                    extraScrambleCount: numExtraSolves,
+                    copies: numCopies
```

Comment:
I realize we didn't have this before, but I'm a fan of trailing commas in these situations (so we don't end up with 2 line diff like we did this time). Could you add a trailing comma after `numCopies`?